src/Components/Options.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import Option1 from './Option1';
+import Option2 from './Option2';
+import './Options.css';
+
+
+export default function Options() {
+  const clicked=()=>{
+    console.log("clicked");
+  }
+  return (
+    <div className='Options' onClick={clicked}>
+        <Option1 />
```

Comment:
I can see that option1 and option2 have same functionality. Reuse one component with parameters. But check if you even need a component to just display an option in navigation